setup.py

```diff
@@ -19,6 +20,9 @@
     "ctapipe-merge = ctapipe.tools.merge:main",
     "ctapipe-fileinfo = ctapipe.tools.fileinfo:main",
     "ctapipe-quickstart = ctapipe.tools.quickstart:main",
+    "ctapipe-ml-train-regressor = ctapipe.ml.tools.train_energy_regressor:main",
+    "ctapipe-ml-train-classifier = ctapipe.ml.tools.train_particle_classifier:main",
```

Comment:
train-energy-regressor
train-particle-classifier

Reply:
yes, particularly as there will eventually be a classifier of the event type (reconstruction quality) that is not the same as the particle type classifier.